include/igl/opengl/ViewerData.cpp

```diff
@@ -80,7 +81,10 @@ IGL_INLINE void igl::opengl::ViewerData::set_mesh(
       Eigen::Vector3d(GOLD_DIFFUSE[0], GOLD_DIFFUSE[1], GOLD_DIFFUSE[2]),
       Eigen::Vector3d(GOLD_SPECULAR[0], GOLD_SPECULAR[1], GOLD_SPECULAR[2]));
 
-    grid_texture();
+    // Alec: is anyone really using this default? It uses trivial UVs and
+    // produces an anisotropic checkerboard...
+    //grid_texture();
```

Comment:
Hmm I think this may be confusing to new users. If I enable textures I'd expect to see a (crappy) checkerboard more than a matcap I think?

Reply:
I can't imagine a circumstance where I want to see nonsense UVs + checkboard by default. Maybe we can trigger checkerboard texture generation if `set_UV` is called and `use_matcap` is (still) false?

Reply:
As said in my latest answer, I think we can kill the `grid_texture()` function, and as default behavior maybe do nothing? That way = no surprise.